lib/jack_and_the_elastic_beanstalk/eb.rb

```diff
@@ -168,7 +168,7 @@ def synchronize_update(timeout: self.timeout)
           end
 
           sleep wait
-          wait = [wait*2, 30].min
+          wait = [wait*2, 120].min
```

Comment:
What does  `[_, 120].min` means?

Reply:
d9cdb59 is related to this implementation. I think it restricted the maximum wait time to `30` (and now I set this `120`). First, wait time is increased by every 2 times, and wait time reaches to limited time, then, from this time, next polling will be executed after `120`.